### DISCUSSION TITLE
Add 'random' URLs to Favorites

### DISCUSSION THREAD

**Discussion by TahomaSoft** [LMS-Community/slimserver#543] [Q&A] [answered]
What is the best way to add a stream URL to the favorites list? Singly, it can be done using the 'Tune In URL' (not to be confused with the TuneIn/RadioTime app) but is awkward. Not sure at all how a batch can be added.

Thanks!

**Comment by michaelherger** [comment, marked as answer]
When you hover over the Favorites item in the default skin's web UI, you'll see a pen button on the right hand side. Clicking this you get in to edit mode for the favourites. This might be the easiest.

There's also an import button in there which allows you to read full OPML files. If you know how to create OPML files.

**Reply by TahomaSoft** [reply]
Thanks!  I will check out both methods.  I tried to import .m3u files, which didn't work so well. I'll try again with OPML files.
